provisioning/miniprov/hare_mp/utils.py

```diff
@@ -349,6 +349,20 @@ def save_ssl_config(self):
         })
         self.kv.kv_put('ssl/hax', ssl_hax)
 
+    '''
+    Provisioner will be generating node_group from each data pod and place
```

Comment:
This doesn't seem very good. I would prefer to have a provisioner exported api instead Hare directly fetching it from Consul based on some mutual understanding. It is always better to have explicit contracts with other components.

Reply:
@mssawant Provisioner only provides URL for GCONF (which will be eventually provided as --config argument)... Apart from this, we do not provide any exported API for any specific logic...

Reply:
Agree, but since Hare does not own this KV, it is better to atleast have a constant corresponding to the kv exported through cortx-utils, like we have for some other conf-store keys, so that if the key changes in future, Hare won't be affected directly.